src/pages/check-email/_icon-grid.tsx

```diff
@@ -38,49 +45,45 @@ const Img = styled.img`
         height: 40px;
     }
 `
-
 const StyledText = styled(Text)`
     @media ${device.tabletL} {
         font-size: 2rem;
     }
 `
 
-const GridContent = [
+const GridContent: TGridContent[] = [
     {
         Icon: Spam,
-        content: localize('The email is in your spam folder (sometimes things get lost there).'),
-        alt: 'Check email in your spam',
+        content: '_t_The email is in your spam folder (sometimes things get lost there)._t_',
+        alt: '_t_Check email in your spam_t_',
     },
     {
         Icon: Typo,
-        content: localize(
-            'The email address you entered had a mistake or typo (happens to the best of us).',
-        ),
-        alt: 'Email ID had a typo error',
+        content:
+            '_t_The email address you entered had a mistake or typo (happens to the best of us)._t_',
+        alt: '_t_Email ID had a typo error_t_',
     },
     {
         Icon: WorkEmail,
-        content: localize(
-            'You accidentally gave us another email address (usually a work or a personal one instead of the one you meant).',
-        ),
-        alt: 'Gave incorrect email ID',
+        content:
+            '_t_You accidentally gave us another email address (usually a work or a personal one instead of the one you meant)._t_',
+        alt: '_t_Gave incorrect email ID_t_',
     },
     {
         Icon: Firewalls,
-        content: localize(
-            'We can’t deliver the email to this address (usually because of firewalls or filtering).',
-        ),
-        alt: 'Firewall filter',
+        content:
+            '_t_We can’t deliver the email to this address (usually because of firewalls or filtering)._t_',
+        alt: '_t_Firewall filter_t_',
     },
 ]
 
 export const IconGrid = () => (
     <Grid>
-        {GridContent.map((item, index) => (
-            <GridCol key={`key-${index}`}>
-                <Img src={item.Icon} alt={item.alt} />
+        {GridContent.map(({ Icon, alt, content }) => (
+            <GridCol key={content}>
```

Comment:
better to add id variable for key or maybe we can use the index instead of content is too big for the key and we cannot ensure its a unique value.

Reply:
It's not a good idea to define id manually, I'll use alt for key since it's shorter and unique!

Reply:
yes ur correct , that's better